apps/site/lib/site_web/controllers/trip_plan_controller.ex

```diff
@@ -69,29 +69,28 @@ defmodule SiteWeb.TripPlanController do
   def to(conn, %{
         "address" => address
       }) do
-    with [latitude, longitude] when is_float(latitude) and is_float(longitude) <-
-           String.split(address, ",") do
+    if String.match?(address, ~r/^(\-?\d+(\.\d+)?),(\-?\d+(\.\d+)?),.*$/) do
+      [latitude, longitude, name] = String.split(address, ",")
```

Comment:
I think we should [use the `parts` option ](https://hexdocs.pm/elixir/1.10.3/String.html#split/3)to limit String.split to always output three items. Otherwise this errors if trying to go to a stop that happens to have a comma in its name (see /stops/111640, it's "Cobbs Corner, Canton")